samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties

```diff
@@ -395,6 +395,7 @@ part_timed_info=If timed, students have until the end of the time limit, or unti
 question_timed_info=If timed, students have until the end of the time limit, or until the Final Submission Deadline, whichever comes first, to submit their work. Their answers will be automatically saved and submitted at the expiration of the timer.
 part_timeSelect_error=Please specify a time limit in hours and/or minutes, or select the "No" option.
 question_timeSelect_error=Please specify a time limit in hours and/or minutes, or select the "No" option.
+multiple_timers_detected=This assessment is configured with multiple kinds of timers. It is recommended to only time either the assessment, parts or questions, because it could be overwhelming for students to have multiple timers on the screen.
```

Comment:
@pablosanroman Would you mind translating it please? Thanks

Reply:
Hello @mpellicer 
question_timed_info=Kronometratua bada, denbora amaitu arte ahal dute bidali ikasleek, edo bata berantiarra izanizan arte, lehenengo gertatzen dena. Erantzunak automatikoki gorde eta bidaliko dira kronometroa amaitzen denean.
part_timeSelect_error=Zehaztu denbora-muga bat edo hautatu "Ez" aukera.
question_timeSelect_error=Zehaztu denbora-muga bat edo hautatu "Ez" aukera.
multiple_timers_detected=Azterketa hau tenporizadore askorekin konfiguratuta dago. Azterketa, parteak edo galderak bakarrik kronometratzea gomendatzen da, pantailan hainbat kronometro izatea gehiegizkoa izan daitekeelako ikasleentzat.